app/src/pages/inside/common/infoLine/attributes/attributesTooltip/attributesTooltip.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import styles from './attributesTooltip.scss';
+
+const cx = classNames.bind(styles);
+
+export const AttributesTooltip = ({ attributes }) => (
+  <div className={cx('attributes-tooltip')}>
+    {attributes.length > 0 ? (
+      attributes.map((item) => (
+        <div key={`${item.key}-${item.value}`}>
+          <span>{item.key}:&nbsp;</span>
+          <span>{item.value}</span>
+        </div>
+      ))
+    ) : (
+      <FormattedMessage id="AttributesTooltip.message" defaultMessage="No attributes" />
```

Comment:
I think it's better to hide the icon if an item doesn't have attributes. To be consistent with grid items.